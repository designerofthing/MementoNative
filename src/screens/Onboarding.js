import React from "react";
import {
  View,
  Text,
  StatusBar,
  StyleSheet,
  Dimensions,
  TouchableOpacity,
  Image,
} from "react-native";
import Swiper from "react-native-swiper/src";
import mori1 from "../../assets/jean-tuttle-memento-mori.jpg";
import mori2 from "../../assets/memento-mori.jpg";
import mori3 from "../../assets/sands-of-time.jpg";
import AppHeader from "../components/AppHeader";

const { width, height } = Dimensions.get("window");

function Onboarding({ navigation, updateAuthState }) {

  return (
    <View style={styles.container}>
      <StatusBar hidden={true} />
      <Swiper>
        <View style={styles.slide}>
          <Image source={mori1} style={styles.image} />
        </View>
        <View style={styles.slide}>
          <Image source={mori2} style={styles.image} />
        </View>
        <View style={styles.slide}>
          <Image source={mori3} style={styles.image} />
        </View>
      </Swiper>
      <View style={styles.textContainer}>
          <AppHeader />
      </View>
      <View style={styles.buttonContainer}>
        <TouchableOpacity onPress={() => navigation.navigate("SignUp")}>
          <View style={styles.signupContainer}>
            <Text style={styles.signup}>Sign up</Text>
          </View>
        </TouchableOpacity>
        <TouchableOpacity onPress={() => navigation.navigate("SignIn")}>
          <View style={styles.loginContainer}>
            <Text style={styles.login}>Log in</Text>
          </View>
        </TouchableOpacity>
      </View>
    </View>
  );
}
export default Onboarding;

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
  slide: {
    flex: 1,
    alignItems: "center",
    justifyContent: "center",
  },
  image: {
    width: width,
    height: height,
  },
  textContainer: {
    position: "absolute",
    bottom: 200,
    marginLeft: 20,
    height: 120,
    alignItems: "flex-start",
    justifyContent: "center",
  },
  titleContainer: {
    width: 300,
    height: 100,
    backgroundColor: "purple",
    alignItems: "center",
    justifyContent: "center",
  },
  title: {
    fontFamily: "serif",
    fontSize: 40,
  },
  subTitleContainer: {
    width: 200,
    height: 50,
    backgroundColor: "purple",
    opacity: 0.7,
    alignItems: "center",
    justifyContent: "center",
  },
  subTitle: {
    fontFamily: "serif",
    fontSize: 24,
  },
  buttonContainer: {
    position: "absolute",
    flexDirection: "row",
    bottom: 70,
    width: width,
    height: 60,
    alignItems: "center",
    justifyContent: "space-around",
  },
  signupContainer: {
    width: 150,
    height: 60,
    backgroundColor: "purple",
    borderRadius: 10,
    alignItems: "center",
    justifyContent: "center",
  },
  signup: {
    fontFamily: "serif",
    fontSize: 20,
    fontWeight: "600",
  },
  loginContainer: {
    width: 150,
    height: 60,
    borderColor: "purple",
    borderRadius: 10,
    borderWidth: 2,
    alignItems: "center",
    justifyContent: "center",
  },
  login: {
    fontFamily: "serif",
    fontSize: 20,
  },
});
