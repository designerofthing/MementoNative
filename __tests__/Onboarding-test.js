import 'react-native';
import React from 'react';
import Onboarding from '../src/screens/Onboarding';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';


jest.setTimeout(15000);

it('renders correctly', async () => {
   const tree = renderer.create(<Onboarding />);
   expect(tree).toMatchSnapshot('Onboarding-test.sj.snap');
});
