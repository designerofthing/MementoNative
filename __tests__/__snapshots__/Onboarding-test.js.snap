// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
          "position": "relative",
        },
        undefined,
      ]
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      autoplay={false}
      autoplayDirection={true}
      autoplayTimeout={2.5}
      bounces={false}
      contentContainerStyle={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          undefined,
        ]
      }
      contentOffset={
        Object {
          "x": 0,
        }
      }
      disableNextButton={false}
      disablePrevButton={false}
      horizontal={true}
      index={0}
      loadMinimal={false}
      loadMinimalSize={1}
      loop={true}
      onIndexChanged={[Function]}
      onMomentumScrollEnd={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      pagingEnabled={true}
      removeClippedSubviews={true}
      scrollsToTop={false}
      showsButtons={false}
      showsHorizontalScrollIndicator={false}
      showsPagination={true}
      showsVerticalScrollIndicator={false}
    >
      <View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              Anyone can create a memento. 
              

               
              

                To get started, click Sign up to create an account and gain access.
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              memento is a memorial space to collect and view images and videos of a friend or loved one that has passed away...
              

               
              

               swipe through to read how it works  
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              Create a memento to preserve the memory of someone special. 
              

               
              

               Then invite their friends and relatives to view and contribute images and videos of that person...
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              The memento creator has control over who can contribute and view the memento. 
              

               
              

               Contributers can view all the images and videos in the person's memento whenever they miss that person...
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              Anyone can create a memento. 
              

               
              

                To get started, click Sign up to create an account and gain access.
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              memento is a memorial space to collect and view images and videos of a friend or loved one that has passed away...
              

               
              

               swipe through to read how it works  
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
    <View
      pointerEvents="none"
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "bottom": 25,
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#007aff",
              "borderRadius": 4,
              "height": 8,
              "marginBottom": 3,
              "marginLeft": 3,
              "marginRight": 3,
              "marginTop": 3,
              "width": 8,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,.2)",
              "borderRadius": 4,
              "height": 8,
              "marginBottom": 3,
              "marginLeft": 3,
              "marginRight": 3,
              "marginTop": 3,
              "width": 8,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,.2)",
              "borderRadius": 4,
              "height": 8,
              "marginBottom": 3,
              "marginLeft": 3,
              "marginRight": 3,
              "marginTop": 3,
              "width": 8,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,.2)",
              "borderRadius": 4,
              "height": 8,
              "marginBottom": 3,
              "marginLeft": 3,
              "marginRight": 3,
              "marginTop": 3,
              "width": 8,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 120,
        "justifyContent": "center",
        "position": "absolute",
        "top": 1,
        "width": 750,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 90,
          "justifyContent": "center",
          "padding": 10,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 60,
            "width": 60,
          }
        }
      />
      <Text
        style={
          Object {
            "color": "#50055E",
            "fontSize": 60,
          }
        }
      >
        memento
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "bottom": 70,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "space-around",
        "position": "absolute",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#50055E",
            "borderRadius": 10,
            "height": 60,
            "justifyContent": "center",
            "width": 150,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 20,
              "fontWeight": "500",
            }
          }
        >
          Sign up
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "borderColor": "#50055E",
            "borderRadius": 10,
            "borderWidth": 2,
            "height": 60,
            "justifyContent": "center",
            "width": 150,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#50055E",
              "fontSize": 20,
              "fontWeight": "500",
            }
          }
        >
          Log in
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`renders correctly: Onboarding-test.sj.snap 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    onLayout={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "transparent",
          "flex": 1,
          "position": "relative",
        },
        undefined,
      ]
    }
  >
    <RCTScrollView
      automaticallyAdjustContentInsets={false}
      autoplay={false}
      autoplayDirection={true}
      autoplayTimeout={2.5}
      bounces={false}
      contentContainerStyle={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          undefined,
        ]
      }
      contentOffset={
        Object {
          "x": 0,
        }
      }
      disableNextButton={false}
      disablePrevButton={false}
      horizontal={true}
      index={0}
      loadMinimal={false}
      loadMinimalSize={1}
      loop={true}
      onIndexChanged={[Function]}
      onMomentumScrollEnd={[Function]}
      onScrollBeginDrag={[Function]}
      onScrollEndDrag={[Function]}
      pagingEnabled={true}
      removeClippedSubviews={true}
      scrollsToTop={false}
      showsButtons={false}
      showsHorizontalScrollIndicator={false}
      showsPagination={true}
      showsVerticalScrollIndicator={false}
    >
      <View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              Anyone can create a memento. 
              

               
              

                To get started, click Sign up to create an account and gain access.
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              memento is a memorial space to collect and view images and videos of a friend or loved one that has passed away...
              

               
              

               swipe through to read how it works  
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              Create a memento to preserve the memory of someone special. 
              

               
              

               Then invite their friends and relatives to view and contribute images and videos of that person...
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              The memento creator has control over who can contribute and view the memento. 
              

               
              

               Contributers can view all the images and videos in the person's memento whenever they miss that person...
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              Anyone can create a memento. 
              

               
              

                To get started, click Sign up to create an account and gain access.
            </Text>
          </View>
        </View>
        <View
          style={
            Array [
              Object {
                "height": 1334,
                "width": 750,
              },
              Object {
                "backgroundColor": "transparent",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "flex": 1,
                "justifyContent": "center",
              }
            }
          >
            <Image
              source={1}
              style={
                Object {
                  "height": 1334,
                  "opacity": 0.5,
                  "width": 750,
                }
              }
            />
            <Text
              style={
                Object {
                  "alignItems": "center",
                  "color": "#50055E",
                  "fontSize": 20,
                  "fontWeight": "600",
                  "height": 300,
                  "justifyContent": "center",
                  "lineHeight": 30,
                  "position": "absolute",
                  "textAlign": "center",
                  "top": 150,
                  "width": 300,
                }
              }
            >
              memento is a memorial space to collect and view images and videos of a friend or loved one that has passed away...
              

               
              

               swipe through to read how it works  
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
    <View
      pointerEvents="none"
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "transparent",
            "bottom": 25,
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "center",
            "left": 0,
            "position": "absolute",
            "right": 0,
          },
          undefined,
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "backgroundColor": "#007aff",
              "borderRadius": 4,
              "height": 8,
              "marginBottom": 3,
              "marginLeft": 3,
              "marginRight": 3,
              "marginTop": 3,
              "width": 8,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,.2)",
              "borderRadius": 4,
              "height": 8,
              "marginBottom": 3,
              "marginLeft": 3,
              "marginRight": 3,
              "marginTop": 3,
              "width": 8,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,.2)",
              "borderRadius": 4,
              "height": 8,
              "marginBottom": 3,
              "marginLeft": 3,
              "marginRight": 3,
              "marginTop": 3,
              "width": 8,
            },
            undefined,
          ]
        }
      />
      <View
        style={
          Array [
            Object {
              "backgroundColor": "rgba(0,0,0,.2)",
              "borderRadius": 4,
              "height": 8,
              "marginBottom": 3,
              "marginLeft": 3,
              "marginRight": 3,
              "marginTop": 3,
              "width": 8,
            },
            undefined,
          ]
        }
      />
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "height": 120,
        "justifyContent": "center",
        "position": "absolute",
        "top": 1,
        "width": 750,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "height": 90,
          "justifyContent": "center",
          "padding": 10,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 60,
            "width": 60,
          }
        }
      />
      <Text
        style={
          Object {
            "color": "#50055E",
            "fontSize": 60,
          }
        }
      >
        memento
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "bottom": 70,
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "space-around",
        "position": "absolute",
        "width": 750,
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#50055E",
            "borderRadius": 10,
            "height": 60,
            "justifyContent": "center",
            "width": 150,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 20,
              "fontWeight": "500",
            }
          }
        >
          Sign up
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "borderColor": "#50055E",
            "borderRadius": 10,
            "borderWidth": 2,
            "height": 60,
            "justifyContent": "center",
            "width": 150,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#50055E",
              "fontSize": 20,
              "fontWeight": "500",
            }
          }
        >
          Log in
        </Text>
      </View>
    </View>
  </View>
</View>
`;
